packages/ui/src/modal/bid/BidModal.tsx

```diff
@@ -570,7 +570,10 @@ export function BidModal({
                       </Button>
                     )}
                     {bidAmount !== '' && hasEnoughWrappedCurrency && (
-                      <Button onClick={placeBid} css={{ width: '100%' }}>
+                      <Button
+                        onClick={() => placeBid({ quantity: 10 })}
```

Comment:
We can remove the hardcoded quantity here, right?

Reply:
yep, sorry about that. I was using it for testing. nice catch